### ISSUE TITLE
Downloading a single page image only

### ISSUE THREAD

**Issue by thadguidry** [internetarchive/openlibrary-client#188]
@tfmorris Hi Tom,

I've run into another little issue.  That of potentially specifying a particular page image to download (rather than downloading a whole zip file).
I noticed that this can be done and found a URL such as this which uses `scale=0` for the maximum resolution that was originally scanned (I think).

https://ia800706.us.archive.org/BookReader/BookReaderImages.php?zip=/29/items/sanfranciscoblue25sanf/sanfranciscoblue25sanf_jp2.zip&file=sanfranciscoblue25sanf_jp2/sanfranciscoblue25sanf_0037.jp2&id=sanfranciscoblue25sanf&scale=0&rotate=0

The key ingredient seems to be the construction of `/sanfranciscoblue25sanf_0037.jp2`  which asks for page 37 ?  But I'm unsure on how the BookReader backend might deal with all this (for instance, is .jp2 always available output from the BookReaderImages?) ... I didn't find documentation, but only played with it in Firefox browser dev tools and watched the network activity GETs.

Any ideas how this can be made more robust or added to the client?  Hmm, and potentially could it be added to the Archive.org Python client you think?

**Comment by tfmorris** [comment]
You are better off asking one of the contributors. I stopped contributing to this library in 2017 because they insisted on a restrictive license for client libraries (#13), unlike the vast majority of open source projects.

I stopped contributing to OpenLibrary in general many months ago due to fundamental disagreement with Internet Archive's values (or rather, lack thereof).

Having said that, it doesn't sound like your question has anything to do with the OpenLibrary client. You might want to have a look at the IIIF endpoint, although I have no idea as to its status. https://iiif.archivelab.org/iiif/documentation

The BookReader API used to be open source, but now is closed source and proprietary to Internet Archive https://github.com/internetarchive/bookreader/issues/255
